soroban-env-common/src/env.rs

```diff
@@ -304,21 +304,13 @@ macro_rules! call_macro_with_all_host_functions {
                 {"0", fn has_contract_data(k:RawVal) -> RawVal}
                 {"1", fn get_contract_data(k:RawVal) -> RawVal}
                 {"2", fn del_contract_data(k:RawVal) -> RawVal}
-                {"3", fn create_contract_from_ed25519(v: Object, salt: Object, key: Object, sig: Object) -> Object}
```

Comment:
I just remembered something - The main reason for allowing contracts to create other contracts using ed25519 was to allow for factory contracts that take the parameters as input to the contract function. This doesn’t work well with `create_contract_from_contract` because another party can frontrun with the same salt but different wasm_hash. I know this scenario is an edge case and the contract creator can always verify the end result before sharing the contract id, but I’m curious what you think about this.